bfd/src/main/java/gov/cms/ab2d/bfd/client/BFDClient.java

```diff
@@ -24,5 +28,7 @@
     Bundle requestPartDEnrolleesFromServer(String contractNumber, int month);
 
     CapabilityStatement capabilityStatement();
+
+    ThreadLocal<String> BFD_BULK_JOB_ID = new ThreadLocal<>();
```

Comment:
Do we really need this or can we just pass in the the value every time we make a call? Also nitpick would be to put this up stop with the other static fields

Reply:
I felt that adding parameters everywhere was too disruptive to the code base. This is a cross-cutting concern, not actually needed by us for the call, more as an aid for BFD to do analysis of their traffic.

" put this up stop " => not sure what you mean.  I didn't want to put it in an Impl class since the callers of the Interface need access.